### ISSUE TITLE
[BUG] Part of prompt ends up in response

### ISSUE THREAD

**Issue by mihirbpi** [ItsPi3141/alpaca-electron#92]
**Describe the bug**
Sometimes when I submit a prompt. Part of the ending of the prompt I submit ends up at the beginning of Alpaca bot's response in the displayed chat messages. The rest of its response is generated as normal.

**To Reproduce**
1. Submit any prompt in the "Enter a message..." submission box. Particularly ones where there are new line.
2. Wait for Alpaca to respond
3. See error

**Expected behavior**
Part of the prompt should not show up in Alpaca's response

**Screenshots**
<img width="714" alt="Screenshot 2023-10-02 at 2 07 56 PM" src="https://github.com/ItsPi3141/alpaca-electron/assets/31811437/93001ce7-35d2-4e15-9d5c-35de7ef9e01a">

**Desktop (please complete the following information):**
 - OS: Mac OSX
 - Version: 13.5.2 (22G91)

**Additional context**
To be fair this is occuring in a fork of the repo I made. I made a modification where the program can get an audip transcription from AWS, put a prompt (which has newlines) based on that description into the submission box, and the user can then submit that prompt. I haven't yet checked if this error occurs in the original repo, however, I think this error should not be caused by the new code in my fork. Here is the link to the forked repo: https://github.com/mihirbpi/alpaca-electron

**Comment by mihirbpi** [comment]
I am also checking if it occurs in the original repo right now

**Comment by mihirbpi** [comment]
Just tried it by entering the prompt manually with the original repo and it also occurs. See below screenshot:
<img width="769" alt="Screenshot 2023-10-02 at 2 32 42 PM" src="https://github.com/ItsPi3141/alpaca-electron/assets/31811437/e7aaf4dc-f530-4510-9dee-17341bdd5dee">

**Comment by ItsPi3141** [comment]
> Just tried it by entering the prompt manually with the original repo and it also occurs. See below screenshot:
> <img width="769" alt="Screenshot 2023-10-02 at 2 32 42 PM" src="https://github.com/ItsPi3141/alpaca-electron/assets/31811437/e7aaf4dc-f530-4510-9dee-17341bdd5dee">
> 

Hi, can you reply with the prompt you used, I want to investigate this further to see if it's a problem with the model, LLaMA.cpp, or my application.

**Comment by mihirbpi** [comment]
Can you please write a summary of the following audio transcription? You should first briefly describe what is happening in the transcription and who is speaking in 1-2 sentences. Then you should summarize the content of the transcription. The summary of the content should be as short as possible while still getting the main ideas across.
Here is the transcription:
Hi, I'm Ray mcdonald on the voice of America. And this is the top five countdown. We're partying with the five most popular songs in the Billboard hot 100 pop singles chart for the week ending February 6th, 2016. The hit list must have made a new year's resolution to give us one new song a week because here it is in fifth place. Alicia Cara jumps a slot with here who supplies that mysterious voice in the song. You'll learn in a moment. Alicia Cara has sold more than 2 million copies of her breakout hit here and here it is in the top five rising one slot to fifth place. Now who provides the mysterious spoken word interludes in the song? The Great Isaac Hayes. That's who it's sampled from his song. Ike's rap two from his classic 1971 album, Black Moses.

> > Just tried it by entering the prompt manually with the original repo and it also occurs. See below screenshot:
> > <img alt="Screenshot 2023-10-02 at 2 32 42 PM" width="769" src="https://user-images.githubusercontent.com/31811437/272093678-e7aaf4dc-f530-4510-9dee-17341bdd5dee.png">
> 
> Hi, can you reply with the prompt you used, I want to investigate this further to see if it's a problem with the model, LLaMA.cpp, or my application.

Can you please write a summary of the following audio transcription? You should first briefly describe what is happening in the transcription and who is speaking in 1-2 sentences. Then you should summarize the content of the transcription. The summary of the content should be as short as possible while still getting the main ideas across.
Here is the transcription:
Hi, I'm Ray mcdonald on the voice of America. And this is the top five countdown. We're partying with the five most popular songs in the Billboard hot 100 pop singles chart for the week ending February 6th, 2016. The hit list must have made a new year's resolution to give us one new song a week because here it is in fifth place. Alicia Cara jumps a slot with here who supplies that mysterious voice in the song. You'll learn in a moment. Alicia Cara has sold more than 2 million copies of her breakout hit here and here it is in the top five rising one slot to fifth place. Now who provides the mysterious spoken word interludes in the song? The Great Isaac Hayes. That's who it's sampled from his song. Ike's rap two from his classic 1971 album, Black Moses.

**Comment by ItsPi3141** [comment]
![image](https://github.com/ItsPi3141/alpaca-electron/assets/90981829/a89c4d3a-8b9d-44d1-b382-4163c008e0b4)

The problem doesn't occur for me, so we can rule out the possibility that it's because of llama.cpp or Alpaca Electron. So I think the problem for you here is the inference parameters. 

I have not had time to create a new release for the latest Alpaca Electron version yet, but you can clone the repository and run the app from the source code. See readme.md for instructions.

**Comment by mihirbpi** [comment]
That's interesting. I did clone and run the latest Alpaca Electron from source and the error I posted occurred with that. However, I used an older version (rolling back to an earlier commit of it) of llama.cpp because the current llama.cpp repo on Github only works with .gguf files and I think Alpaca Electron uses .bin. For me for some reason Alpaca Electron wasn't working with the latest llama.cpp. It could be that my model is the problem. Is that what you mean by the inference parameters?
